dashboard/src/app/(landing)/components/openSourceCallout.tsx

```diff
@@ -0,0 +1,33 @@
+import { Github } from 'lucide-react';
+import { Button } from '@/components/ui/button';
+import Link from 'next/link';
+
+export function OpenSourceCallout() {
+  return (
+    <section className='py-20'>
+      <div className='container mx-auto px-4 sm:px-6 lg:px-8'>
+        <div className='text-center'>
+          <Github className='text-primary mx-auto mb-6 h-16 w-16' />
+          <h2 className='mb-4 text-3xl font-bold sm:text-4xl'>
+            <span className='text-blue-600 dark:text-blue-400'>Open Source</span> & Community Driven
```

Comment:
Question: I don't know the best practices - but should the "&" be replaced with "&amp;"? 🤔